packages/x-components/tests/e2e/cucumber/common-steps.spec.ts

```diff
@@ -34,6 +58,44 @@ Then('no identifier results are displayed', () => {
   cy.getByDataTest('identifier-result-item').should('not.exist');
 });
 
+// Facets
+When(
+  'filter number {int} is clicked in facet {string}',
+  (filterNumber: number, facetName: string) => {
+    clickFacetNthFilter(facetName, filterNumber);
+  }
+);
+
+Then(
+  'search request contains selected filter number {int} is {boolean}',
+  function (this: any, simpleFilter: number, isContained: boolean) {
+    cy.wait('@requestWithFilter')
+      .its('request.body')
+      .then(requestBody => {
+        if (isContained) {
+          expect(requestBody).to.contain(this['clickedFilter' + simpleFilter.toString()]);
+        } else {
+          expect(requestBody).not.to.contain(this['clickedFilter' + simpleFilter.toString()]);
+        }
+      });
+  }
+);
+
+Then(
+  'clicked filter {int} is selected in facet {string} is {boolean}',
```

Comment:
I don't know if the statement here is clear.  Are we trying to say that the statement 'clicked filter {int} is selected in facet {string}' is true or false?

I would say that something like `selection status of filter number {int} in facet {string} is {boolean}` makes the message clearer.